inscriptions/ISic000128.xml

```diff
@@ -192,22 +204,21 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#TA">
                 <ab>
-                    <lb n="1"/><expan><abbr>P</abbr><ex>ublius</ex></expan> <g ref="#interpunct">·</g> Antistius <g ref="#interpunct">·</g> <expan><abbr>P</abbr><ex>ubli</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ilius</ex></expan>
+                    <lb n="1"/><persName type="attested"><expan><abbr>P</abbr><ex>ublius</ex></expan></persName> <g ref="#interpunct">·</g> <persName type="attested">Antistius</persName> <g ref="#interpunct">·</g> <persName type="attested"><expan><abbr>P</abbr><ex>ubli</ex></expan></persName> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ilius</ex></expan>
```

Comment:
NB `<persName>` goes once around the complete name, so `<persName type="attested">Publius Antisitius Publii filius Claudia (tribu) Cavarianus</persName>`. The patronym is then tagged with persName internally to that, as recording a separate person. Each onomastic element gets `<name>` around it, and we put type="patronym" on name element around the Publi in this case. We are not bothering to encode the others, but you can see the principles at https://github.com/ISicily/ISicily/wiki/Encoding:-text#named-entities and could encode praenomen, cognomen, etc. if you wanted to... Note too the full formally correct encoding for the tribal...